### DISCUSSION TITLE
xxxxxx

### DISCUSSION THREAD

**Discussion by NSIOISN** [XTLS/Xray-core#2910] [Q&A]
XXX

**Comment by zxbiao** [comment]
封禁是看封禁到哪一步。
最简单就是DNS污染（抢答）
二是网管直接将可疑的加密连接且目标不是国内的IP加入屏蔽（黑洞）列表
还有一个可能性就是客户机装有合法的后门软件监测（如，IPGuard及自签的SSL证书做中间人攻击）
